README.md

```diff
@@ -3,16 +3,27 @@ Example CNF
 
 [![PR Validation](https://github.com/openshift-kni/example-cnf/actions/workflows/push.yaml/badge.svg)](https://github.com/openshift-kni/example-cnf/actions/workflows/push.yaml)
 
-Example CNF is an OpenShift workload to exercice an SRIOV setup.
+Example CNF is an OpenShift workload to exercice an SRIOV setup, such as the following:
 
 ![Schema](documentation/schema.png)
 
 It is providing the following operators:
 
-* testpmd-operator
 * trex-operator
+    * It provides TRex Traffic Generator, decomposed in two components: a TRex server, deployed as `trexconfig-<x>` pod, which takes care of deploying and configuring a TRex server instance, and a TRex application, deplayed as `trex-app` job, that starts TRex server, generating traffic towards the system under test.
```

Comment:
it could be nice to speak about TRex CR, pinning the config, and TRex operator, ensuring reconciliation via controller-manager pod by running short-living Trex jobs.